.circleci/main.yml

```diff
@@ -562,6 +577,26 @@ jobs:
       - store_artifacts:
           path: /tmp/debug-dump
           destination: debug-dump
+  create-nightly-tag:
```

Comment:
You might want to consider moving the `create-nightly-tag` job to to config.yml, that way, you can figure out if you just need to run that earlier in the process, and you may not need the downstream variables (`run_ci`, etc)

Reply:
good call. okay take a look made it simpler. will test on a test repo i have.